Kochbuch/src/app/rezeptanlegen/rezeptanlegen.component.html

```diff
@@ -0,0 +1,141 @@
7ShCZuC/yIzMqQ4vpgtojhpo9XqkKS2ULXy1UGeJm130bbZsrZ3MbrLdFfN0wwA1cDl9VXu9VSMa4ujyFcDX29yjPbrv7atpWazNQHGNpxZ01MKzWOEgsJdHUDTa0shDZCdR0klORrmRnifc2lwNUjXEkCuJx55p+3+qbW70Mkwe80pmrVbD+oGPCutxuV+Db8tcdaIrrwDhQeA0lWaKV/d488Qtg3j+NK6SUAvINCcfhy92a0N9bCdgYDQV4Ks3ebesPdciU8QFpeUQo1oaHkKYiX8n8iUSDOqkwyGMaQqwFpiRcGUFbSEOOILiXl9MOZgaC4AaFQyryx2idpbUgkBRLy8eyMuAJPADmrydntvbx3OtSbcuw7Eg7cZSI0+020yXZL2pz2ziZamkVbKiaqKifGpGMldyRNLtb5TqNWkg6Qa4NwPlzzPwWMMNxNG0zOPcB93jTn7lC94364bPcmo3iHIjcJp56Tcg0rUAlJW47+WFEhVCSQa+GK61211wQImnUSPKfkMafkoOqSM6XD3qBXG/2eP+lzJF8CEySXmHUB1yRH6dOmpVaaSrVVPq8eGLWKynFQG4g0IySdI1poUM/wDiZAnbyQpiV+pTpR0plsNhbrqOClrCKkgAVUSeAqcaiy225Y0SEHDOv6rkLpmunFOE6I81JksvICHUOKWTyzzqPiOGGwQVJacwuxCiNxhKfjS4/RS8xJQpD8dXyqBFD94OH6TC8HKmRXF7MDTIqF2uwWjbRRFtTRZaWUuy3SSpxa6/KVHMhPIHE64u5LrzSGvD28VB7bYhRowTvvaze+s8GfLvCA1ObX7i3lvWtKBVajTOuqvChGOe33BZMQBU1rWvHLFPMLnAUOfABUCm3F237mnOWe7vS0tuPBhRbLaVE0CUlIIBGQFaU549aigEls0SsAwFcarm6BjWUpTqj92ZkvI7i3G6y7SxbXv0VLjuhSlaHHdCVFJUon1cwagcsZH1M0f8e2Njy4a6e4Y8uC0npdjXXBLTUAH8FaGRuQJNUucOaT/ZjAMsa8F6GwIxdtIc++RHtyXKYIu1rc8GStSwHZT4oosspOdACNSuA4DPh0Ft2fO40A4c/ctJs20G8kAINPbirqdq3Y257jBfkRmRBhyWjaduw2giOpaFekrQk/mOLVQVJrzri92ki5e1zqUBwaBhXw4kq43+Jm0wOih8tQauP3YjnwAV3VdpttXFC7vvq2Nypr7heagJecQ201pybe6agFmvGn3nGyuNlti3uXLRUmtKnLkaZrxl2+TB3btnHSBStBieYrkvhdtu7OiOxNuW2BZY3ylqEy2zqA+UqKQCr7ScZq93mHbmObDpaOgA8F0ZayXTg6Qlx6mqjEjeceUh0qAfJoepqISKZklROfwxln762ckkV640+PFWLbBzOiC+5NyrdlOri/mUrUqJSyBjM3l458pdGK//AFVzbwBraO/VRBncchshzqpkD94BJzrx00x2td1lipqIcPA19yfJbtdksrluG2x7ZNvMt5uLFt0dcmZJcXRKGm0lS1FXgACTjSvYzcIwWDzcPyUDV2CS4+XiqHb/APrJ25f9u3629p3Jl23Elfs4MtLGhthLzWtNw0LzLSSaAqA9XLxsYPTTrd7XXZDWZ0/uPFvisvuvqyKOBxhrqqQOlP3IJxrX3l3yuwCdMS9ftty4y7q40whKa6lSFuOEHT11NIVqCdI0gZVxJG7WkLqtbSM1ocfbT+Kytpu19fXUZkIo0jpTqacaKxe0rPEfubM2+tJfcDwV7J0BSUJB+UjhiRC866vXqctAKNV4bb3MsVitzbVsiRbc202EssspQk1y/dSmmeL2XeREyraABQGWHcd5iSrsbJt1ql7ftl4lOs3J2XGRKffcSD0lLTq6SKnICvPjjUWDIpIWyOIcSKknh06LJ7hPK2Z0batANABx6pu3Fu+2xQI8VthJaKglpY01yyOWWY4DFNuW8QNGltKjgfqpNnYSE1cSgluOeLon0thND6qV5+X92MJudoy9GQWltCYULZV0fgFaZKytSKlgngB4n4Yxd1bvtHUdjy9uivI9MgwQ43J3DaiQ1R2FB597JCgqigfHLwx1gvWshwoXHJJ8dCha1uu7yZKlrkqa0nUVuKJ1fbWtcWu3htNUhxUOZ5yaFN2pFr3hCVbrq4j3qWqQbjUddpw8KK4rTXilX4HHW7jtbxuh5FeB4j25JkesYhV8kXq5bR3ROt1zDaDAcS3VtSiVpIrroRQAggpp9uM42zdaSCmYz8OilF2oI4QN5R5dpejuLDodQC0hSiAajnTP7RjWwSd6Ag8QoL8HVTntt5xD0W12+2PXK4yhVqIyhbq1V5pSipOfPDrK3/j6YmMJcfeucjgRUmgCe9+wu4u3mI0qPtqRZGnQRImSEAJqflQCapST/NiVudnJHR0jC3qofd1CkZBVFO7167vXWLIiSY0qLZ3iWnFRXA+5XjVwt5JSfLEK2bAHAuNSOBVLeC4eDqwb0xWf07fRdc++Wzu4u5brdntsWW1x+hte6LYCmblcykqWy84r1hpuqQ4W/VVQzyIxphc6IXXLqhrBnwrUYD8FmIttE+GRJ+Pj0Q+j/TX+g3mJHm3RUJu0Mri3iCpOqSZKU6FdKQKAeqh9SaEAeOIQ3Zs7C44h2I5eBC0FjtJtpWyMNKChBxr4FQu623cOybj7Zy4JuMJRPtpob0E0/dWAclUxCe2C4rQUPJaljiEUbFZe7dysJ3Ratozndvtp1/q71I7K05VU2XikuJz4oCsUF06wgnEMkobIcm/u6YDmpEcsbnhpcBVeMbZ7k7muLMPcFyNqsRBMn2byQEJH8WYWskcBkMaCARRtpiT1z+CumyWtuyrRV3UewCk7+3NnbRZ1lRmtt6VPTXX9KAPUFJPSKCCnLjq4+WJUrzHGHNaXGuQ5LlHuNxcPoDT3LbtvudAsF6iL23cRYEyV9GZdkKU680wo6XSgEjUKZjTxpxxGNyR5g0t+
```

Comment:
Bitte die Datei in `src/assets` ablegen, ich weiß, im bestehenden Code befinden sich Base64 Bilder, die wollen wir aber auch noch da rausbekommen